src/main/java/com/google/sps/data/Post.java

```diff
@@ -43,9 +63,35 @@
   private String description = "";
   private String collegeId = "";
   private int timeSort = 0;
+  private int rank = 0;
 
   public boolean valid = true; // If false, the post shouldn't be saved - it might have malicious data.
 
+  /* Constructor for Post testing */
+  public Post(String organizationName, int month, int day, int startHour, 
```

Comment:
In general, we discourage exposing constructors just for testing. There is currently no way to ensure that only test classes use this constructor. Consider adding a @VisibileForTesting annotation (I'm not 100% sure that this works for constructors). This way, you can add a lint (out of the scope of this project) that will ensure that this is only called in a test class. 

Your alternative is to construct the Post using the requestToPost. I think the second option is better

Reply:
Gotcha, implementing the tests again using requestToPost!